core/src/main/java/org/lflang/generator/GeneratorBase.java

```diff
@@ -211,10 +192,6 @@ private void createMainInstantiation() {
    */
   public void doGenerate(Resource resource, LFGeneratorContext context) {
 
-    // FIXME: the signature can be reduced to only take context.
-    // The constructor also need not take a file config because this is tied to the context as well.
-    cleanIfNeeded(context);
```

Comment:
What happened here?

Reply:
The `cleanIfNeeded(context)` method call is relocated to `LFGenerator.java`, so that the `model-gen` folder can be cleaned too before verification is run.
https://github.com/lf-lang/lingua-franca/blob/baa5061c6297b989a6d26beb728cd368b20b36c0/core/src/main/java/org/lflang/generator/LFGenerator.java#L120-L121